napari/layers/points/points.py

```diff
@@ -471,6 +540,45 @@ def properties(self, properties: Dict[str, np.ndarray]):
             self._face_color_property = ''
             warnings.warn('property used for face_color dropped')
 
+        if self._edge_color_property and (
+            self._edge_color_property not in self._properties
+        ):
+            self._edge_color_property = ''
+            warnings.warn('property used for edge_color dropped')
+
+    @property
+    def current_properties(self):
+        """Dict[str: np.ndarray]: properties for the next added point."""
+        return self._current_properties
+
+    @current_properties.setter
+    def current_properties(self, current_properties):
+        self._current_properties = current_properties
+
+        if (
+            self._update_properties
+            and len(self.selected_data) > 0
+            and self._mode != Mode.ADD
+        ):
+            props = self.properties
+            for k in props:
+                props[k][self.selected_data] = current_properties[k]
+            self.properties = props
+
+            self.refresh_colors()
+        self.events.current_properties()
+
+    @property
+    def default_properties(self):
+        """dict {str: array (N,)}, DataFrame: default annotations"""
```

Comment:
As above... Let's aim for consistency in typing and docstrings!

Reply:
As I understand it, we will never actually have a dataframe here, right? Also, dataframes can't have different-length columns, so it would not be the right model for this data.

Reply:
> As above... Let's aim for consistency in typing and docstrings!
Fixed!


> Also, dataframes can't have different-length columns, so it would not be the right model for this data.

Good point. Fixed.